### DISCUSSION TITLE
Strum app?

### DISCUSSION THREAD

**Discussion by ezzz01** [djphazer/O_C-Phazerville#58] [Q&A]
Hi, 

in this article [1], CDM mentions a new Strum app being included in the 1.7 release. But I can't find anything about it neither in the docs, nor in the module with the newest version installed. Is it a mistake on the CDM's part?

[1] https://cdm.link/2024/02/ornament-crime-open-module-gets-new-powers-in-phazerville-1-7/

**Comment by djphazer** [comment]
Very cool, I wasn't aware of this article! Looks like they're citing things from the Beta release notes...

Strum was included in v1.7-beta1 for testing, but it still needs some work. Watch for it to reappear in the next release! (or on a development branch if you're impatient 😉)
